Library/Homebrew/dev-cmd/vendor-gems.rb

```diff
@@ -61,6 +61,9 @@ def vendor_gems
         %w[
           mechanize
         ].each do |gem|
+          (HOMEBREW_LIBRARY_PATH/"vendor/gems").cd do
+            Pathname.glob("#{gem}-*/").each { |path| FileUtils.rm_r(path) }
```

Comment:
Does `GEM_PATH=#{HOMEBREW_LIBRARY_PATH}/vendor gem cleanup` work any nicer?

Reply:
Per DM, I can't figure out the incantation to make this work